network/p2p/peerstore/peerstore.go

```diff
@@ -58,7 +58,7 @@ type addressData struct {
 	mu           *deadlock.RWMutex
 
 	// role is the role that this address serves.
-	role phonebook.PhoneBookEntryRoles
+	role phonebook.RoleSet
```

Comment:
```suggestion
	roles phonebook.RoleSet
```
And change the comment to match the name and plural form.

Reply:
applied here and in phonebook